### ISSUE TITLE
`/transactions/count/{id}` fails with "no such route"

### ISSUE THREAD

**Issue by davidyuk** [aeternity/ae_mdw#1854]
The paths I've tried
https://mainnet.aeternity.io/mdw/v3/transactions/count/ak_2afpJzgHcx9ckQ3FwQngK51f8A9hTCX4tSpf6u8jzGqFaoqorf
https://mainnet.aeternity.io/mdw/v3/transactions/count/dcg.chain
https://mainnet.aeternity.io/mdw/v3/transactions/count/ct_1RiXDD3BxVsw5x14JhwCq5hPy55LZ2xN4KBL8kUHbArEDhVNS
https://mainnet.aeternity.io/mdw/v3/transactions/count/ok_qJZPXvWPC7G9kFVEqNjj9NAmwMsQcpRu6E3SSCvCQuwfqpMtN
in all cases it fails with 404, "no such route", I expect to get transactions counts instead 🤷‍♀️

**Comment by sborrazas** [comment]
You need to use `/v3/transactions/count?id=ak_...` instead for this one

**Comment by davidyuk** [comment]
It really works this way! So, the issue is in open api definition, `id` is a path parameter there
```json
    "/transactions/count/{id}": {
      "get": {
        "deprecated": false,
        "description": "Get transactions count and its type for given aeternity ID.",
        "operationId": "GetTransactionsCountById",
        "parameters": [
          {
            "description": "The ID of the address/name/oracle/etc",
            "in": "path",
            "name": "id",
            "required": true,
            "schema": {
              "type": "string",
              "x-example": "ak_g5vQK6beY3vsTJHH7KBusesyzq9WMdEYorF8VyvZURXTjLnxT"
            }
          }
        ],
        "responses": {
          "200": {
...
```

**Comment by vatanasov** [comment]
Solved in #1878 , docs: https://mainnet.aeternity.io/mdw/swagger/#/default/GetAccountTransactionsCount
